pipeline_steps/common.groovy

```diff
@@ -278,13 +278,54 @@ def archive_artifacts(){
   } finally{
     // still worth trying to archiveArtifacts even if some part of
     // artifact collection failed.
-    archiveArtifacts(
-      allowEmptyArchive: true,
-      artifacts: "artifacts_${env.BUILD_TAG}.tar.bz2,artifact_index.txt"
-    )
+    stash(
+      name: "artifacts",
+      includes: "artifacts_${env.BUILD_TAG}.tar.bz2")
```

Comment:
The venvPlaybook methods that we use to execute ansible has an assumption that it's running from the CIT node, which is a different OS than the created slave nodes. But it's probably better that I make that method more general rather than having to pass the file between nodes, so I'll make that update.